### ISSUE TITLE
Document how to run unit tests

### ISSUE THREAD

**Issue by stefnotch** [nagisa/rust_tracy_client#117]
It'd be lovely if potential contributors could run the unit tests that rely on loom. Currently, I do not know how to run those.

`RUSTFLAGS="--cfg loom" cargo test --release` does not seem to work

**Comment by nagisa** [comment]
Yeah, unfortunately the test suite sucks quite a bit due to the fact that they rely on tracy to operate and mocking this component out and/or testing with it is Extremely Inconvenient -- it really isn't meant to be used in automated contexts like these.

Maybe one day I'll figure out some work-around to this but today is not that day :/
